pytorch_lightning/accelerators/ddp2_accelerator.py

```diff
@@ -61,21 +61,23 @@ def train(self):
         return self.ddp_train(process_idx=self.task_idx, mp_queue=None, model=model)
 
     def training_step(self, args):
+        return self._step(args)
+
+    def validation_step(self, args):
+        return self._step(args)
```

Comment:
```suggestion
    def validation_step(self, *args):
        return self._step(*args)
```
or can we define what is the type? and how about `**kwargs`?

Reply:
I think I'll leave this for a separate PR as I don't want to mess with the accelerator logic too much... kwargs are not explicitly passed into the training step upstream, so this needs to be an involved change